packages/common/src/ui/components/index.ts

```diff
@@ -16,6 +16,7 @@ import Tooltip from '@mui/material/Tooltip';
 import Radio from '@mui/material/Radio';
 import RadioGroup from '@mui/material/RadioGroup';
 import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
```

Comment:
there is a small version of this in one of Manu's designs.. so I was hoping to add one in at some point, was going to use it for something else.
Really liked how clear the radio buttons were, though I think what you've done is better - nice and clean looking.